multiqc/utils/search_patterns.yaml

```diff
@@ -199,6 +199,26 @@ ccs/v4:
 ccs/v5:
   contents: '"id": "ccs_processing"'
   fn: "*.json"
+checkatlas/summary:
+  fn: "*_checkatlas_summ.tsv"
+checkatlas/adata:
+  fn: "*_checkatlas_adata.tsv"
+checkatlas/qc:
+  fn: "*_checkatlas_qc.tsv"
+checkatlas/qc_fig:
+  fn: "*_checkatlas_qc.png"
+checkatlas/umap:
+  fn: "*_checkatlas_umap.png"
+checkatlas/tsne:
+  fn: "*_checkatlas_tsne.png"
```

Comment:
I almost never merge modules that include images into reports.

MultiQC doesn't know how many samples it will need to handle, and as such every module should work with 1-1000 samples. With images, you can't have more than a couple before the report is unusable. Worse, the file size will bloat the HTML file and it will crash the browser surprisingly fast. It's not accessible and the data cannot be exported into `multiqc_data` for downstream use.

Because of this, MultiQC ignores any image files by default, even if they are defined as a search pattern here. So this won't work as it stands (I think).

Please look into whether these plots can be _recreated_ within MultiQC by parsing the raw data and generating plots instead.

Reply:
They quite certainly can be created using MultiQC, however I have a question about scalability here: Those UMAP/tSNE plots are scatterplots that can easily have 100k+ points. Is this something the MultiQC plotting enging can still handle decently, especially if there are multiple such plots included in the report? 

Second, for those plots being readable, the point size needs to be somewhat dynamic depending on the total number of cells, reducing the point size if there are more cells. For instance, scanpy uses the heuristic `120000 / n_cells` for these plots. Is this something that can be handled in MultiQC?

Reply:
The images and html report generation have been moved to checkatlas and custom content of multiqc.